storage/cloud-client/storage_set_public_access_prevention_inherited.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Please add a descriptive top-level comment (go/code-snippets-style#descriptive-top-level-comment)

Reply:
Added one in, let me know if this is what is expected, I just searched around in the python samples until I found an example of where someone else is doing this (doesn't look like we have been really doing any of these in storage/cloud-client)

Reply:
Np, this is great.  Can you please move the comment so that it is above the regionalization tag "# [START storage_set_public_access_prevention_inherited]"

Reply:
done. Are these conventions well documented? Noticing some inconsistency throughout our docs here.

Reply:
Ah I do see the link you shared says to have it outside the bounds of the region tag. Looks like we haven't followed that everywhere in our samples thus far.

Reply:
I am checking with a Technical Writer to get some guidance, to make sure I don't lead you astray. I will update soon.

Reply:
It looks to me like you're on point with your recommendations, which makes sense to me. I just noticed that some samples, such as those in `storage/transfer-service` place it inside of the tag, which leads to things like here having it displayed in the documentation, which may be not desirable.
https://cloud.google.com/storage-transfer/docs/create-manage-transfer-program

Reply:
Good catch.  Can you make an issue on this repo to document/track, and assign me?